styles/itch.io/catppuccin.user.less

```
@@ -0,0 +1,622 @@
+        background-color: @subtext0;
+      }
+
+      .jam_filtered_jam_calendar_widget .jam_calendar_filters > .label {
+        color: @text;
+      }
+    }
+
+    .blog_post_grid_widget .blog_post .post_type > span,
+    .blog_post_grid_widget .blog_post .post_type > a {
+      background-color: @red;
+      color: @base;
+    }
+
+    .page_tabs .tab.active, .page_tabs .tab:hover {
+      box-shadow: inset 0 -2px 0 0 var(--itchio_link_color, @accent);
+    }
+
+    .register_form_widget .validated {
+      &.valid input {
+        border-color: @green;
+      }
+      .icon {
+        color: @green;
+      }
+
+      &.invalid input {
+        border-color: @red;
+      }
+      .input_error_message {
+        color: @red;
+      }
+    }
+
+    .events_page .event_row {
+      border-color: @surface1;
+
+      .event_header .action_btn:not(.disabled):hover {
+        color: @text;
+      }
+
+      .button_row .outline_button:hover {
+        background-color: @accent;
+        color: @crust;
+        border-color: @accent;
+      }
+    }
+
+    .feed_page .right_column .mega_notification {
+      background: @red;
+      color: @crust;
+    }
+
+    .tooltip_drop .tooltip {
+      box-shadow: 0 0 0 2px @accent;
+
+      &::after {
+        filter: @accent-filter;
+      }
+    }
+
+    .dashboard_page .header_notification {
+      background-color: fade(@accent, 20%);
+      color: @accent;
+    }
+
+    /* Creator Dashboard Analytics */
+    .graph_container .label circle,
+    .graph_container .popup_trigger .dot,
+    .graph_container .graph path {
+      stroke: @accent;
+    }
+
+    /* featured bundle */
+    .sale_list_widget .sale_row {
+      border-color: @surface2;
+    }
+
+    .sale_list_widget .sale_games .sale_gradient {
+      background-image: linear-gradient(
+        to right,
+        rgba(255, 255, 255, 0),
+        @base
+      );
+    }
+
+    .browse_filter_group_widget .toggle_more_btn:hover {
+      background-color: @surface0;
+      color: var(--itchio_body_color_med);
+    }
+
+    .selectize-input, .selectize-control.single .selectize-input.input-active {
+      background-color: @base;
+    }
+
+    .selectize-control.single .selectize-input::after {
+      border-color: @overlay0 transparent transparent transparent;
+    }
+
+    .selectize-control.single .selectize-input, .selectize-dropdown.single {
+      border-color: @surface2;
+    }
+
+    .selectize-control.single .selectize-input {
+      background-image: linear-gradient(to bottom, @base, @base);
+      box-shadow: 0 0;
+    }
+
+    .selectize-dropdown {
+      background-color: @base;
+      border-color: @surface2;
+
+      .active {
+        color: @text;
+        background-color: @surface0;
+      }
+
+      .create {
+        color: @text;
+      }
+    }
+
+    .selectize-dropdown, .selectize-input, .selectize-input input {
+      color: @text;
+    }
+
+    .selectize-input .dropdown-active::before {
```

```suggestion
    .selectize-input.dropdown-active::before {
      background-color: @surface2;
    }
```
it was just a space 💀